cgi/display.pl

```diff
@@ -117,6 +117,13 @@
 	return Apache2::Const::OK;
 }
 
+if ($request_ref->{rate_limiter_blocking} eq 1) {
```

Comment:
```suggestion
if ($request_ref->{rate_limiter_blocking} ) {
```

eq is for string comparison